.github/workflows/publish.yml

```diff
@@ -36,20 +39,20 @@ jobs:
 
   publish-docs:
     name: Publish documentation
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3.1.0
         with:
           fetch-depth: 0
 
+      - name: Cache Coursier cache
+        uses: coursier/cache-action@v6.4.0
+
       - name: Set up JDK 11
-        uses: olafurpg/setup-scala@v10
+        uses: coursier/setup-action@v1.3.0
         with:
-          java-version: adopt@1.11
-
-      - name: Cache Coursier cache
-        uses: coursier/cache-action@v6.2
+          jvm: adopt:11
```

Comment:
@ennru @johanandren I know this didn't change, but in my opinion we should use an exact jdk version for the publish workflows in the spirit of reproducible builds.